legion/engine/physics/physics_statics.hpp

```diff
@@ -227,6 +229,8 @@ namespace legion::physics
 
         static std::tuple< math::vec3, math::vec3> ConstructAABBFromTransformedVertices(const std::vector<math::vec3>& vertices,const math::mat4& transform);
 
+        static std::tuple<math::vec3, math::vec3> CombineAABB(const std::tuple<math::vec3, math::vec3>& first, const std::tuple<math::vec3, math::vec3>& second);
+
```

Comment:
this file should really have comments, but at least add them for new methods you create